src/cljs/snapshot/lumo/repl.cljs

```diff
@@ -811,6 +811,19 @@
   (deps/index-upstream-foreign-libs))
 
 ;; --------------------
+;; Introspection
+
+(defn ^:export get-arglists
+  "Return the argument lists for the given symbol as string."
+  [s]
+  (when-let [var (some->> s repl-read-string first (resolve-var @env/*compiler*))]
+    (if-not (:macro var)
+      (:arglists var)
+      ;; filtering &env &form for macros
+      (map #(filterv (complement '#{&env &form}) %)
```

Comment:
I had to do this because apparently in the real repl that symbols have &env and `&form as arglists, while in the tests they don't. I know that I fixed a symptom and I haven't investigated the cause but hopefully we'll figure it out at some point.